client/my-sites/site-settings/podcasting-details/style.scss

```diff
@@ -172,3 +151,8 @@
 		margin: 12px 0 6px;
 	}
 }
+
+a.podcasting-details__support-link {
```

Comment:
Let's not rely on this being an `a` tag.  The class name is specific enough to cover only this component, and the implementation of the `ExternalLink` component does sometimes return a `button` instead (IIRC) and its implementation may change further in the future.

Reply:
The `a` in the selector was necessary to get the link underlined on the details page. I will create a separate issue to address this minor visual difference.